zulip/zulip/__init__.py

```diff
@@ -823,7 +823,8 @@ def do_register() -> Tuple[str, int]:
 
             for event in res["events"]:
                 last_event_id = max(last_event_id, int(event["id"]))
-                callback(event)
+                if event["type"] != "heartbeat":
+                    callback(event)
```

Comment:
I think I'd prefer an early return style -- doing a `continue` if it is that type; it'll be nicer when this loop inevitably starts doing some new thing in addition to calling the callback.